.github/workflows/site.yml

```diff
@@ -96,7 +107,11 @@ jobs:
           SITE="./checkstyle/target/site"
           LINK="https://${{env.AWS_BUCKET_NAME}}.s3.${{env.AWS_REGION}}.amazonaws.com"
           aws s3 cp $SITE s3://${{env.AWS_BUCKET_NAME}}/$FOLDER/ --recursive
-          echo $LINK/$FOLDER/index.html > message
```

Comment:
it is better to keep link to main page and add link to specific section as second line in message.

Reply:
> Have you tested these changes? Please share results

yes i tried `act` but in short if i tell you error is `act -l` is only reading `codeql-analysis.yml` and not other yml files and even in that showing few errors which are mostly indentation based.